.golangci.yml

```diff
@@ -20,11 +20,6 @@ linters:
   settings:
     depguard:
       rules:
-        use_modern_packages:
-          list-mode: lax
-          deny:
-            - pkg: math/rand$
-              desc: Please use math/rand/v2
```

Comment:
@timvaillancourt Is this now handled through some other way? We still want to deny usage of `math/rand` in favor of `math/rand/v2`?